### PR TITLE
variables module

### PR DESCRIPTION
**Pull request by DaryaMak**
Scope:
Module 2 tasks are done

### CODE REVIEW

File: tasks/variables/variables.js [hunk 1]
```diff
@@ -6,44 +6,60 @@
...
  * TODO create variable named str2 and set value equal str plus " Dou"
  */
-
+let str2 = str + " Dou";
+console.log(str2);
 /**
  * TODO create variable named bool and set value of equality comparison of int and int2 variables
  */
-
+let bool = int == int2;
```
Comment (SnezhanaMatskevich) [inline tasks/variables/variables.js]: Here should be "==="

File: tasks/variables/variables.js [hunk 2]
```diff
@@ -6,44 +6,60 @@
...
+};
+console.log(human);
 
 /**
  * TODO set new key of human object named lastName and value "Dou"
  */
-
+let var2 = "Dou"
+human['lastName'] = var2;
+console.log(human);
```
Comment (SnezhanaMatskevich) [inline tasks/variables/variables.js:55]: The easiest way to do it:
`human.lastName = 'Dou';`

File: tasks/variables/variables.js [hunk 3]
```diff
@@ -6,44 +6,60 @@
...
 /**
  * TODO set new key of human object named fullName and value of concatenation of firstName and lastName values
  */
-
+let var3 = `${human.firstName} ${human.lastName}`;
+human['fullName'] = var3;
+console.log(human);
```
Comment (SnezhanaMatskevich) [inline tasks/variables/variables.js:60]: `human.fullName = human.firstName + " " + human.lastName;`

File: tasks/variables/variables.js (hunk 1 above)
Comment (DaryaMak) [reply tasks/variables/variables.js]: Thanks for noticing, I've changed it

File: tasks/variables/variables.js (hunk 2 above)
Comment (DaryaMak) [reply tasks/variables/variables.js:55]: Thanks, I've changed it

File: tasks/variables/variables.js (hunk 3 above)
Comment (DaryaMak) [reply tasks/variables/variables.js:60]: Thanks for the advice, I've picked it up

**Pull request merged**